src/Tizen.NUI/src/public/WebView/WebView.cs

```diff
@@ -114,7 +114,7 @@ public WebView(string locale, string timezoneId) : this(Interop.WebView.New2(loc
         /// Creates a WebView with an args list.
         /// <param name="args">args array. The first value of array must be program's name.</param>
```

Comment:
param should not be inside \<summary>
You could add that those params are passed to WebKit

Reply:
Done.